xdsl/ir.py

```diff
@@ -990,6 +990,16 @@ def last_op(self) -> Operation | None:
         """The last operation in `self`"""
         return self.ops[-1] if len(self.ops) else None
 
+    def insert_op_after(self, new_op: Operation,
```

Comment:
I think this is a bit redundant compared to what
https://github.com/xdslproject/xdsl/pull/758/files#diff-295d7b48e29fb49bba3333242ac92a2d04d38f3bbcf46d7ab4d2474a3e4957c3R1024
offers